modules/gdscript/gdscript_parser.cpp

```diff
@@ -2413,7 +2579,20 @@ void GDScriptParser::_parse_block(BlockNode *p_block, bool p_static) {
 		}
 
 		switch (token) {
-
+			case GDScriptTokenizer::TK_EOF:
+			case GDScriptTokenizer::TK_ERROR:
+			case GDScriptTokenizer::TK_NEWLINE:
+			case GDScriptTokenizer::TK_CF_PASS: {
+				// will check later
+			} break;
+			default: {
+				if (p_block->has_return) {
+					_set_error("Unreacheable code.");
```

Comment:
Is this really necessary? Unreachable code might not be an error, I use it a lot when debugging, it can be very annoying that I cannot run the script for that reason. It also has the potential of breaking many projects (though I admit, having a check is nice)

Reply:
Yeah, I was going to remove this (and eventually make a warning) but I forgot. It shouldn't really stop the code from running.